<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="LAPTOP-644D0OPQ" failures="1" tests="8" name="Default test" time="1017.583" errors="0" timestamp="2021-12-26T22:40:20 IST">
  <testcase classname="com.TestNG_RunnerClass.TestNgRunner" name="women" time="150.903"/>
  <testcase classname="com.TestNG_RunnerClass.TestNgRunner" name="dresses" time="80.526">
    <skipped/>
  </testcase> <!-- dresses -->
  <testcase classname="com.TestNG_RunnerClass.TestNgRunner" name="dresses" time="80.518">
    <skipped/>
  </testcase> <!-- dresses -->
  <testcase classname="com.TestNG_RunnerClass.TestNgRunner" name="dresses" time="80.518">
    <skipped/>
  </testcase> <!-- dresses -->
  <testcase classname="com.TestNG_RunnerClass.TestNgRunner" name="dresses" time="80.514">
    <failure type="org.testng.internal.thread.ThreadTimeoutException" message="Method com.TestNG_RunnerClass.TestNgRunner.dresses() didn&amp;apos;t finish within the time-out 80510">
      <![CDATA[org.testng.internal.thread.ThreadTimeoutException: Method com.TestNG_RunnerClass.TestNgRunner.dresses() didn't finish within the time-out 80510
at org.testng.internal.MethodInvocationHelper.invokeWithTimeoutWithNewExecutor(MethodInvocationHelper.java:384)
at org.testng.internal.MethodInvocationHelper.invokeWithTimeout(MethodInvocationHelper.java:283)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:604)
at org.testng.internal.TestInvoker.retryFailed(TestInvoker.java:213)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:58)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- dresses -->
  <testcase classname="com.TestNG_RunnerClass.TestNgRunner" name="blouses" time="87.456"/>
  <testcase classname="com.TestNG_RunnerClass.TestNgRunner" name="blouses" time="97.63"/>
  <testcase classname="com.TestNG_RunnerClass.TestNgRunner" name="womenTshirt" time="97.864"/>
  <testcase classname="com.TestNG_RunnerClass.TestNgRunner" name="summerDresses" time="0.0">
    <ignored/>
  </testcase> <!-- summerDresses -->
</testsuite> <!-- Default test -->
